apps/opik-backend/src/main/resources/liquibase/db-app-analytics/migrations/000009_extend_feedback_source_type.sql

```diff
@@ -0,0 +1,7 @@
+--liquibase formatted sql
+--changeset DanielAugusto:000009_extend_feedback_source_type
+
+ALTER TABLE ${ANALYTICS_DB_DATABASE_NAME}.feedback_scores
+    MODIFY COLUMN `source` Enum8('sdk', 'ui', 'online_scoring');
```

Comment:
I believe this has to be merged separately (Before this PR), because if any rollback is necessary this will block it